### PR TITLE
Add region options

### PR DESCRIPTION
**Pull request by LittleSheep2Code**
I add a region option, let different region can get fast speed to access server.
Example:

```
translate('Hello!', {to: 'en', region: 'cn'})
      .then(console.log);
```

PS. I delete my fork repo, so I recreate a pull request XD

### CODE REVIEW

**Comment by ZiuChen** [comment]
`google-translate-api-cn` may you need.

File: index.js [hunk 1]
```diff
@@ -25,12 +25,13 @@ function translate(text, opts) {
 
     opts.from = opts.from || 'auto';
     opts.to = opts.to || 'en';
+    opts.region = opts.region || 'com';
```
Comment (1RB) [inline index.js:28]: The opts.region option has been added to the function parameters and a default value of 'com' is set.

File: index.js [hunk 2]
```diff
@@ -25,12 +25,13 @@ function translate(text, opts) {
...
 
     opts.from = languages.getCode(opts.from);
     opts.to = languages.getCode(opts.to);
 
     return token.get(text).then(function (token) {
-        var url = 'https://translate.google.com/translate_a/single';
+        var url = 'https://translate.google.' + opts.region + '/translate_a/single';
```
Comment (1RB) [inline index.js:34]: The URL for the Google Translate API has been updated to include the region option.

**Review by 1RB — APPROVED** [review]
The code change looks good overall.
The opts.region option has been added to the function parameters and a default value of 'com' is set.
The URL for the Google Translate API has been updated to include the region option.
No bugs have been detected in this code change.

**Comment by LittleSheep2Code** [comment]
I think this change is useless now. Because google just stop service in china mainland. So translate.google.cn domain is doesn't work. And in other country, I think they didn't need to access google's other domain. Just use the google.com is OK.

**Pull request closed without merging**